backend/config/db/initial-queries.sql

```diff
@@ -1,10 +1,40 @@
-CREATE TABLE "users"(
-  "id" uuid PRIMARY KEY,
-  "username" varchar(255) UNIQUE,
-  "password" varchar(100),
-  "profile_picture" varchar(255),
-  "display_name" varchar(100),
-  "bio" varchar(255),
-  "interests" varchar(255),
-  "github" varchar(100)
-)
+CREATE TABLE users (
+    id uuid NOT NULL PRIMARY KEY UNIQUE,
+    username character varying(255),
+    password character varying(100),
+    profile_picture character varying(255),
+    display_name character varying(100),
+    bio character varying(255),
+    interests character varying(255),
+    github character varying(100)
+);
+
+--/////////////////////////////////////////////////////////////
+
+CREATE TABLE projects (
+    project_id uuid NOT NULL PRIMARY KEY,
+    project_owner_user_id uuid REFERENCES users(id),
+    project_title character varying(1000),
+    project_description character varying(10000),
+    creation_timestamp timestamp without time zone
+);
+
+
+--/////////////////////////////////////////////////////////////
+
+CREATE TABLE posts (
+    post_id uuid NOT NULL PRIMARY KEY,
+    project_id uuid REFERENCES projects(project_id),
+    post_creator_user_id uuid REFERENCES users(id),
+    post_content character varying(1000),
+    creation_timestamp timestamp without time zone
+);
+
+--/////////////////////////////////////////////////////////////
+
+CREATE TABLE project_collaborator_xref (
```

Comment:
It's a cross-reference table to enable many-to-many relations between users (collaborators) and projects table. Since 1 project can have 1+ collaborators (user) and 1 user can collaborate on 1+ projects, this is to maintain the 1 record per user in user table and 1 record per project in project table granularity.

This can be used to get all collaborators of a project.

| UserId  | ProjectId  |
|---|---|
| UserA  | ProjectA   |
| UserB  | ProjectA  |
| UserA | ProjectB  |
| UserC | ProjectB  |